experimental/dds/tree2/src/domains/leafDomain.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SchemaBuilder } from "../feature-libraries";
+import { ValueSchema } from "../core";
+
+/**
+ * Names in this domain follow https://en.wikipedia.org/wiki/Reverse_domain_name_notation, and are versioned.
+ */
+const builder = new SchemaBuilder("Leaf");
+
+/**
+ * @alpha
+ */
+export const number = builder.leaf("fluidframework.com.leaf.1.number", ValueSchema.Number);
+/**
+ * @alpha
+ */
+export const boolean = builder.leaf("fluidframework.com.leaf.1.boolean", ValueSchema.Boolean);
+/**
+ * @alpha
+ */
+export const string = builder.leaf("fluidframework.com.leaf.1.string", ValueSchema.String);
+/**
+ * @alpha
+ */
+export const handle = builder.leaf("fluidframework.com.leaf.1.handle", ValueSchema.FluidHandle);
+
+/**
+ * @alpha
+ */
+export const primitives = [number, boolean, string] as const;
+
+/**
+ * Types allowed as roots of Json content.
+ * @alpha
+ */
+export const all = [primitives, ...primitives] as const;
```

Comment:
Should this include "handle"?

Reply:
(Or should "primitives" include "handle"?)

Reply:
handles aren't comparable with === either, which is part of the normal JavaScript primitive meaning, at lest in my mental model.